app/controllers/users.py

```diff
@@ -79,6 +81,18 @@ def post(self):
             template,
             subject
         )
+        
+        # check if user exists in anonymous table and assign them to a project
+        anonymous_user_exists = AnonymousUser.find_first(email=email)
+
+        if anonymous_user_exists:
+            project = Project.get_by_id(anonymous_user_exists.project_id)
+            user_details = User.find_first(email=email)
+            
+            new_role = ProjectUser(role=anonymous_user_exists.role, user_id=user_details.id)
+            project.users.append(new_role)
+            saved_project_user = project.save()
+            anonymous_user_exists.delete()
```

Comment:
Also a check here to make sure the delete was successful i.e **deleted = anonymous_user_exists.delete()**, **if not deleted:**

Reply:
otherwise everything else looks good.

Reply:
This has been resolved in latest commit.